app/views/layouts/application.html.erb

```diff
@@ -28,7 +28,7 @@
     <%= image_tag current_user.avatar_url, size: User::AVATAR_SIZES[:small], class: "right-link" if current_user.avatar? %>
     <span class="right-link username"><%= current_user.name %></span><span class="right-link">Signed in as: </span>
     <% else %>
-    <%= link_to 'Sign in', sign_in_path %>
+    <%= link_to 'Sign in', sign_in_path, class:"hidden" %>
```

Comment:
Hmmm not sure about this way of ensuring the tests still have something to click on :P 

Would be better if the tests were updated to click on the new, real 'sign in' button instead of this invisible old one.